app/rtc_s1.py

```diff
@@ -583,6 +701,29 @@ def run(cfg):
                            ' from this ISCE3 version. The sub-swath masking'
                            ' was disabled.')
 
+        # Calculate layover shadow mask when requested
+        if save_layover_shadow_mask:
+            layover_shadow_mask_file = (f'{bursts_output_dir}/{product_id}'
+               f'_layover_shadow_mask.{imagery_extension}')
+            calculate_layover_shadow_mask(burst,
+                                geogrid,
+                                cfg.dem,
+                                layover_shadow_mask_file,
+                                threshold_rdr2geo=cfg.rdr2geo_params.threshold,
+                                numiter_rdr2geo=cfg.rdr2geo_params.numiter,
+                                threshold_geo2rdr=cfg.geo2rdr_params.threshold,
+                                numiter_geo2rdr=cfg.geo2rdr_params.numiter)
+            
+            logger.info(f'file saved: {layover_shadow_mask_file}')
```

Comment:
Maybe we can just print `file saved...` if it's not temporary, i.e., if `not flag_bursts_files_are_temporary`.